cedar-policy/src/api.rs

```diff
@@ -2124,19 +1557,10 @@ impl EntityUid {
     pub fn from_json(json: serde_json::Value) -> Result<Self, impl miette::Diagnostic> {
         let parsed: cedar_policy_core::entities::EntityUidJson = serde_json::from_value(json)?;
         // INVARIANT: There is no way to write down the unspecified entityuid
-        Ok::<Self, cedar_policy_core::entities::JsonDeserializationError>(Self(
+        Ok::<Self, cedar_policy_core::entities::JsonDeserializationError>(Self::new(
             parsed.into_euid(|| JsonDeserializationErrorContext::EntityUid)?,
         ))
     }
-
-    /// Testing utility for creating `EntityUids` a bit easier
-    #[cfg(test)]
-    pub(crate) fn from_strs(typename: &str, id: &str) -> Self {
-        Self::from_type_name_and_id(
-            EntityTypeName::from_str(typename).unwrap(),
-            EntityId::from_str(id).unwrap(),
-        )
-    }
 }
 
 impl FromStr for EntityUid {
```

Comment:
Should we move this `FromStr` impl too, since we moved the definition of `EntityUid`?